definitions/tools/mutect.cwl

```diff
@@ -2,75 +2,66 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-label: "mutect2 (GATK 3.6)"
-baseCommand: ["/usr/bin/java", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "MuTect2"]
+label: "Mutect2 (GATK 4)"
+
+baseCommand: ["/bin/bash", "Mutect2.sh"]
+
 requirements:
     - class: ResourceRequirement
       ramMin: 20000
       tmpdirMin: 100000
     - class: DockerRequirement
-      dockerPull: "mgibio/cle:v1.3.1"
+      dockerPull: "broadinstitute/gatk:4.1.2.0"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'Mutect2.sh'
+        entry: |
+            set -o pipefail
+            set -o errexit
+
+            /gatk/gatk Mutect2 -O $1 -R $2 -I $3 -tumor $5 -I $4 -normal $6 -L $7
+            gunzip mutect.vcf.gz
+            sed "s/\t$5\t/\tTUMOR\t/g; s/\t$6/\tNORMAL/g" mutect.vcf > mutect_final.vcf
+            bgzip mutect_final.vcf
+            tabix mutect_final.vcf.gz
+            mv mutect.vcf.gz.stats mutect_final.vcf.gz.stats
+            /gatk/gatk FilterMutectCalls -R $2 -V mutect_final.vcf.gz -O mutect_final_filtered.vcf.gz
```

Comment:
It looks like FilterMutectCalls has lots of options that we may want to expose at some point, but the defaults seem reasonable for now.